### DISCUSSION TITLE
Homebrew arm and x86 compatibility

### DISCUSSION THREAD

**Discussion by kadorito** [Whisky-App/Whisky#390] [Q&A] [answered]
Hello

I would love to start using whisky with the final Sonoma release, to stop messing with 2 versions of brew. My question : does whisky require brew x86 to be installed ? If no, does it work with an existing brew arm installed on the machine ?

Thanks

**Comment by IsaacMarovitz** [comment, marked as answer]
Whisky does not require a brew install either x86 or ARM.

**Reply by kadorito** [reply]
Just to be 100% sure, you mean whisky including gptk or just whisky ? Thanks

**Reply by IsaacMarovitz** [reply]
Whisky ships with precompiled versions of GPTK Wine, ergo, you don't need brew.

**Reply by kadorito** [reply]
Thats' the definition of awesome :-)
